<?xml version="1.0"?>
<launch>

<rosparam command="load" file="$(find panda)/config/2Dparams.yaml" />

<node name="Remote" pkg="panda" type="Remote" output="log"/>


<node name="Agent_0" pkg="panda" type="Agent_node" output="screen">
  <param name="i_id" value="0"/>
  <param name="robot_type" value = "panda"/>
</node>

</launch>